### PR TITLE
Update build scripts

### PR DESCRIPTION
**Pull request by kshlm**
Updates build scripts and rpm specfile. Docker build updates will be done in a seperate PR.

### CODE REVIEW

**Comment by kshlm** [comment]
retest this please

**Comment by prashanthpai** [comment]
retest this please

File: extras/rpms/glusterd2.spec [hunk 1]
```diff
@@ -35,34 +36,25 @@ Preview release of the next generation GlusterFS management framework and daemon
...
 mkdir -p src/%(dirname %{import_path})
 ln -s ../../../ src/%{import_path}
 
 pushd src/%{import_path}
 # Build glusterd2
-make glusterd2
-make glustercli
+%{gd2make} glusterd2
+%{gd2make} glustercli
+%{gd2make} glusterd2.toml
```
Comment (prashanthpai) [inline extras/rpms/glusterd2.spec]: Just a question: make a conf file, what does this do ?

Comment (kshlm) [reply extras/rpms/glusterd2.spec]: Generates glusterd2.toml with proper paths set for statedir, logdir etc.

**Pull request merged**